cinema/management/commands/wait_for_db.py

```
@@ -0,0 +1,20 @@
+import time
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    help = "Waits for the database to be available"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+                db_conn.cursor()
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)
```

Optional robustness improvement: you catch `OperationalError`, which is appropriate for most DB connection failures. If you want broader coverage for other driver exceptions while still surfacing unexpected errors, you could either also catch other known DB driver exceptions or add a guarded generic exception handler that re-raises unexpected exceptions after logging them. Avoid silently swallowing all exceptions.